### PR TITLE
DPC-85 add location and lastupdated (if set) to headers

### PR DESCRIPTION
**Pull request by whuang85**
**Why**

To align with FHIR, adding location and last modified headers

**What Changed**

Added the location and last modified date (when available) to the bundle

**Choices Made**

Did not change the code for organization to have last modified date as that table doesn't contain those columns so it's a heavier lift.

**Tickets closed**:

https://jira.cms.gov/browse/DPC-85

**Future Work**

List any additional tickets that have either been created due to work in this PR, or existing tickets that expand upon the feature or provide additional fixes.

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-api/src/test/java/gov/cms/dpc/api/APITestHelpers.java [hunk 1]
```diff
@@ -199,4 +198,33 @@ private static Validator provideValidator(InjectingConstraintValidatorFactory fa
                 .configure().constraintValidatorFactory(factory)
                 .buildValidatorFactory().getValidator();
     }
+
+    public static Practitioner createPractitionerResource(String NPI, String orgID) {
```
Comment (embh) [inline dpc-api/src/test/java/gov/cms/dpc/api/APITestHelpers.java:202]:
```suggestion
    public static Practitioner createPractitionerResource(String npi, String orgID) {
```
Would you be okay with making this and the MBI parameter in `createPatientResource()` lowercase? Might get mistaken for constants if someone is scanning the code quickly.

File: dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/PatientResourceTest.java [hunk 2]
```diff
@@ -245,4 +244,33 @@ void testCreateInvalidPatient() throws IOException, URISyntaxException {
...
+        IGenericClient client = APIAuthHelpers.buildAuthenticatedClient(ctx, getBaseURL(), ORGANIZATION_TOKEN, PUBLIC_KEY_ID, PRIVATE_KEY);
+        Patient patient = APITestHelpers.createPatientResource("4S41C00AA00", APITestHelpers.ORGANIZATION_ID);
+
+        MethodOutcome methodOutcome = client.create()
+                .resource(patient)
+                .encodedJson()
+                .execute();
+
+        String location = methodOutcome.getResponseHeaders().get("location").get(0);
+        String date = methodOutcome.getResponseHeaders().get("last-modified").get(0);
```
Comment (embh) [inline dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/PatientResourceTest.java:259]: Do these need to be `Location` and `Last-Modified`? Or is it case insensitive?

Comment (whuang85) [reply dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/PatientResourceTest.java:259]: > Do these need to be `Location` and `Last-Modified`? Or is it case insensitive?

@embh I went with the capitalized version originally but for some reason, the IGenericClient returns lower case from the response headers.

Comment (embh) [reply dpc-api/src/test/java/gov/cms/dpc/api/resources/v1/PatientResourceTest.java:259]: Oh weird! That's good to know. 👍

**Pull request merged**